test/ci-operator-integration/expected_files/expected_with_template.json

```diff
@@ -1151,21 +1151,57 @@
         ]
       },
       {
-        "apiVersion": "v1",
```

Comment:
Let's commit a PR first that adds a `jq --sort-keys` step to the test so that we don't have a diff here on ordering.

Reply:
I will prepare a PR for it. In the meanwhile this is the diff from the ordered files:
```
1167d1166
<           "creationTimestamp": null,
1251d1249
<               "resources": {},
1368d1365
<               "resources": {},
1425d1421
<               "resources": {},
1468d1463
<               "resources": {},
1495,1496c1490
<         },
<         "status": {}
---
>         }

```

Reply:
PR: #153